src/main/java/Todo.java

```diff
@@ -0,0 +1,20 @@
+public class Todo extends Task{
+    private final String title;
+
+    public Todo(String command) throws DukeException {
+        if (command.isEmpty()) {
+            throw new DukeException("Please use this format: Todo <Activity>");
+        }
+        this.title = command;
+        System.out.println("added: " + this.toString());
+    }
+
+    public String toString(){
```

Comment:
There is some inconsistency with the white spacing (for e.g. there should be a white space between "toString()" and '{' according to Java Coding Standard